css/fonts.css

```diff
@@ -0,0 +1,37 @@
+@font-face {
+  font-family: 
+  "Montserrat";
+  src: 
+  url("../fonts/Montserrat-Bold.woff2") format("woff2"),
+  url("../fonts/Montserrat-Bold.woff") format("woff");
+  font-weight: 
+  bold;
+  font-style: 
+  normal;
+  font-display: 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5705021)